### PR TITLE
minor bin script fixes

### PR DESCRIPTION
**Pull request by justin1121**
Also add some more files to the .dockerignore file.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+.mypy_cache/
+.pytest_cache/
+
+docs/build
+dist/
+tmp/
+**/*.pb
+data/
+*.egg-info
+*.so
```
Comment (mortendahl) [inline .dockerignore]: some repetition here

File: bin/serve [hunk 2]
```diff
@@ -89,13 +90,13 @@ tfe.set_tfe_trace_flag(enable_traces)
...
 
 remote_config = tfe.RemoteConfig(
-    {
-        'master': master_address,
-        'server0': server0_address,
-        'server1': server1_address,
-        'crypto-producer': producer_address,
-        'weights-provider': provider_address
-    }
+    OrderedDict([
```
Comment (mortendahl) [inline bin/serve]: 👍 you could probably also use 
```python
OrderedDict(
  master=...,
  server0=..., )
```

Comment (justin1121) [reply bin/serve]: Much nicer! Thanks.

Comment (justin1121) [reply bin/serve]: although its doesn't like names like `crypto-producer` in the argument list. Will leave it how it is for now.

Comment (justin1121) [reply bin/serve]: Doens't like the `-` character.

**Pull request merged**